src/main/resources/application.yml

```diff
@@ -11,7 +11,7 @@ spring:
     show-sql: true
 
 server:
-  port: 8090
+  port: 8092
 
```

Comment:
What't the need for changing a port number?

Reply:
when i am trying to launch the spring boot application it shows port not 8090 is busy so i had changed that to 8092 so that my application could start normally but now i would change back to 8090